frontends/election-manager/src/config/types.ts

```diff
@@ -103,7 +104,13 @@ export type OptionalFile = Optional<File>;
 // Cast Vote Records
 export interface CastVoteRecord
   extends Dictionary<
-    string | string[] | boolean | number | number[] | BallotLocale
+    | string
+    | string[]
+    | boolean
+    | number
+    | number[]
+    | BallotLocale
+    | Adjudication[]
```

Comment:
Ugh. I think we should stop using a `Dictionary` for this. Not for this PR, though.